search-app/test/cmr/search/test/api/routes.clj

```diff
@@ -10,23 +11,23 @@
 (deftest find-concept-id-and-revision-id-from-path-w-extension
   (testing "concept-id"
     (are [path-w-extension concept-id]
-         (= concept-id (r/path-w-extension->concept-id path-w-extension))
+         (= concept-id (core-api/path-w-extension->concept-id path-w-extension))
 
          "C1-PROV1" "C1-PROV1"
          "C1-PROV1.xml" "C1-PROV1"
          "C1-PROV1/3.echo10" "C1-PROV1")
 
    (testing "revision-id"
      (are [path-w-extension revision-id]
-          (= revision-id (r/path-w-extension->revision-id path-w-extension))
+          (= revision-id (core-api/path-w-extension->revision-id path-w-extension))
 
           "C1-PROV1/2" 2
           "C1-PROV1/3.echo10" 3))))
 
 (deftest get-search-results-format-test
   (testing "format from headers"
     (are [path headers default-mime-type expected-format]
-         (=  expected-format (#'r/get-search-results-format :collection path headers default-mime-type))
+         (=  expected-format (core-api/get-search-results-format :collection path headers default-mime-type))
```

Comment:
Are these the tests that are failing now in CI? Is there somewhere in the tests where we try to redefine that function? Just trying to figure out why the original `#'` was used.

Reply:
The original `#'` is used to call a private function in the qualified namespace. We are not using this enough. For functions that is only invoked outside of its defined namespace in tests, the functions should be `private`. I have refrained myself from giving people actions on this sort of things during review though.

Reply:
What Liu said ... except I feel that we might make too many things private; there's a lot of duplicated private code in the CMR ... those sorts of things should be put into an appropriate namespace for common use.

Note, however, that the use of `#'` really is abuse of Clojure's (mostly)private support. We need to make sure that only tests are using that to circumvent function privacy. There is an argument to be made that if you want to test a function, it might be better to have it be a part of your declared, public testing support "API" ...